praw/__init__.py

```diff
@@ -238,6 +238,13 @@ def config_boolean(item):
         self.https_proxy = (obj.get('https_proxy') or
                             os.getenv('https_proxy') or None)
         # We use `get(...) or None` because `get` may return an empty string
+
```

Comment:
I thought that i'd need to check because I was attempting to make it completely optional?  If it's not there, it will set true, otherwise it will attempt to configure it how the user requested.